standard/mock/mock_lpdb.lua

```diff
@@ -0,0 +1,97 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Mock/Lpdb
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Json = require('Module:Json')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local lpdbData = Lua.import('Module:Mock/Lpdb/Data', {requireDevIfEnabled = true})
+
+local mockLpdb = {}
+
+local _lpdb = {
+	lpdb = mw.ext.LiquipediaDB.lpdb
+}
+
+function mockLpdb.setUp()
+	mw.ext.LiquipediaDB.lpdb = mockLpdb.query
+end
+
+function mockLpdb.tearDown()
```

Comment:
Maybe?

```suggestion
function mockLpdb.dispose()
```

Reply:
Dispose would to me indicate it's automatically called when garbage collected. while `tearDown` would match the `setup`/`tearDown` usually used in Unit Tests to indicate functions to be run before and after tests, respectively.

https://developer.apple.com/documentation/xctest/xctestcase/set_up_and_tear_down_state_in_your_tests
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

Reply:
You're right!